### PR TITLE
upgrades

### PR DESCRIPTION
**Pull request by itsMapleLeaf**
Upgrades the things

### CODE REVIEW

**Comment by itsMapleLeaf** [comment]
for some reason, I get a 500 on `/themes/` and `/integrations/` and I can't figure out why 🤔 

https://astro-build-l4a0uvz83-astrodotbuild.vercel.app/themes
https://astro-build-l4a0uvz83-astrodotbuild.vercel.app/integrations

**Comment by itsMapleLeaf** [comment]
> for some reason, I get a 500 on `/themes/` and `/integrations/` and I can't figure out why 🤔
> 
> [astro-build-l4a0uvz83-astrodotbuild.vercel.app/themes](https://astro-build-l4a0uvz83-astrodotbuild.vercel.app/themes) [astro-build-l4a0uvz83-astrodotbuild.vercel.app/integrations](https://astro-build-l4a0uvz83-astrodotbuild.vercel.app/integrations)

Turns out the `functionsPerRoute: false` probably fixed this... I guess maybe we'll try taking that out in another PR 🤷

File: src/content/blog/images.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ coverImage: "/src/content/blog/_images/images/blog-hero-image-updates.webp"
 socialImage: "/src/content/blog/_images/images/og-image-image-better.jpg"
 ---
 
-import { Image } from "@astrojs/image/components"
+import { Image } from "astro:assets"
```
Comment (tony-sull) [inline src/content/blog/images.mdx:13]: Revert this one, looks like a find/replace caught the blog post's example of the old integration

**Review by tony-sull — APPROVED** [review]
Very nice!  Just one tiny note to revert a content change in the images blog post

File: src/content/blog/images.mdx (hunk 1 above)
Comment (itsMapleLeaf) [reply src/content/blog/images.mdx:13]: This one was a functional import, there were other ones _elsewhere_ that I updated instead

**Pull request merged**